### PR TITLE
Task list and submit and pay combined

### PR DESCRIPTION
**Pull request by shashisk97**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-523
https://tools.hmcts.net/jira/browse/PRL-444

### Change description ###
Addition of task list tab and submit and pay screens

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by yogendraU — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -75,3 +75,76 @@ uk:
...
+  s2s-auth:
+    url: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    microservice: prl_cos_api
+    totp-secret: GT2V6PLQ34XGPTPM
+  api:
+    url: https://idam-api.aat.platform.hmcts.net
+  client:
+    id: prl-cos-api
+    redirect_uri: https://manage-case.aat.platform.hmcts.net/oauth2/callback
+    secret: W7supXKMJgcEWKBS
```
Comment (swaroopa86) [inline src/main/resources/application.yaml:113]: secrets need to replaced

File: src/integrationTest/resources/application.properties [hunk 2]
```diff
@@ -2,3 +2,53 @@ env=${test_environment:local}
...
+logging.level.org.springframework.web=DEBUG
+
+###############################################
+#  IDAM                                       #
+###############################################
+auth.idam.client.baseUrl=https://idam-api.aat.platform.hmcts.net
+auth.idam.client.redirectUri=https://manage-case.aat.platform.hmcts.net/oauth2/callback
+auth.idam.client.secret=DBssSOq0KKLNBf2z
+auth.idam.client.clientId=xuiwebapp
+
```
Comment (swaroopa86) [inline src/integrationTest/resources/application.properties:20]: secrets need to be replaced

File: src/main/resources/application.yaml (hunk 1 above)
Comment (yogendraU) [reply src/main/resources/application.yaml:113]: we will sort this once we have secrets pulled from azure

**Review by swaroopa86 — APPROVED** [review]

File: src/integrationTest/resources/application.properties (hunk 2 above)
Comment (yogendraU) [reply src/integrationTest/resources/application.properties:20]: we will sort this once we have secrets pulled from azure

**Review by BharadwajSairam — APPROVED** [review]

**Pull request merged**